docs/development/manufacturer/fc_documentation/how-to-create-board-documentation.mdx

```diff
@@ -86,12 +85,20 @@ Images should be added to the `<TARGETNAME>-images` directory, and referenced in
 
 Top/bottom images should be added for each board, and any additional images or diagrams can be added as needed. The images can be in any web-compatible format, but `.png` is recommended.
 
+Use an image in a square aspect ratio (1:1) for the first image in the document, and use consistent filetype naming for the images - for example `.JPG` and `.jpg` can cause conflicts and confusion.
```

Comment:
```suggestion
Use an image in a square aspect ratio (1:1) for the first image in the document, and use consistent filetype naming for the images - for example `.jpg` can cause conflicts and confusion.
```

Reply:
the main suggestion is to not mix uppercase and lowercase filenames, not to avoid `jpg` images completely

Reply:
Updated suggestion